include/mysqli.php

```diff
@@ -181,11 +181,6 @@ function db_create_database($database, $charset='utf8',
 function db_query($query, $logError=true, $buffered=true) {
     global $ost, $__db;
 
-    if ($__db->unbuffered_result) {
-        $__db->unbuffered_result->free();
```

Comment:
It's freed by the ORM when the cursor reaches the end of the result set. So the double free is part of the issue. This was only used by the export system for database backup. It should be rewritten to use the ORM...